toc.md

```diff
@@ -824,7 +826,17 @@ We also decided to compare the results of JRuby vs MRI. Because jRuby can run in
 
 ## Conclusion
 
-It was a lot of fun to work on this project. We learned a lot about threaded code, memory usage by different types of OS entities, Redis data structures and other Ruby interpreters. We really hope our findings will help other developers in their journey through the pathways of threaded code.
+It was a lot of fun to work on this project! Our goal was to share potentially useful knowledge with the community and to introduce background job processors to people unfamiliar with them. After all it's usually a very interesting enterprise to understand how something works under the hood!
+
+As for us, we learned and expanded our knowledge about:
+
+* concurrency, and specifically threads
+* parallel execution using multiple processes and other Ruby interpreters
+* lower level details about threads and processes, such as their memory footprint
+* the impact on computational and memory resources when using multiple threads and multiple processes
+* using Redis as a key:value store and taking advantage of its awesome features such as native data structures, great performance and snapshots
+* building a Ruby gem
+* providing users with an easy-to-use yet powerful CLI
 
 While our first version works, we are far from done. There are still a handful of features we'd like to add in the near future. Some of these faeatures are:
```

Comment:
same comment as before, I thing this line should be changed to somehting more like:
> Workerholic being in its alpha stage we do not advise to use this gem in a production environment. As for the next steps we have a few features in mind: ...

This way we're not devaluating our work by saying negative things like "we're far from done".